cluster/common.sh

```diff
@@ -455,7 +466,7 @@ EOF
       exit 2
   esac
 
-  popd
+  popd "$@"
```

Comment:
What shellcheck failure are you addressing?

Reply:
In ./cluster/common.sh line 471:
  popd
  ^--^ SC2119: Use popd "$@" if function's $1 should mean script's $1.